src/main/java/be/kakumi/kachat/utils/ChatManager.java

```diff
@@ -20,15 +28,85 @@
      * @param player Player who sent the message
      * @return List of available players
      */
-    List<Player> getValidReceivers(Channel channel, Player player);
+    public final List<Player> getValidReceivers(Channel channel, Player player) {
```

Comment:
Now it's final so it's not overriden, as requested in the javadocs of the method